docs/userguide/src/SUMMARY.md

```diff
@@ -36,6 +38,8 @@
     - [Performance Tuning](portingguide/perf_tuning/prefix.md)
         - [Link Time Optimization](portingguide/perf_tuning/lto.md)
         - [Optimizing Allocation](portingguide/perf_tuning/alloc.md)
+    - [VM-specific Concerns](portingguide/concerns/prefix.md)
```

Comment:
In the step-by-step porting guide I preferred to use the term "language runtime" or "runtime" instead of VM since MMTk is more general. I won't comment on all the instances here. Just doing it once.

Reply:
I agree.  But MMTk seems to use the term "VM" extensively, like "VM binding" instead of "runtime binding".  I think it may be a legacy from JikesRVM, where MMTk is just a component of the VM.

Personally, I think "runtime" is more accurate to describe the thing the Rust MMTk supports.  Not all languages that have GC use VM.  Go, RPython (not PyPy), GHC, ScalaNative and KotlinNative are a few examples.

I don't plan to change every occurrence of "VM" with "runtime" in the documentation now.  If we do, we should do the same for the code, too.

Reply:
I mean the point was that we don't add more documentation that uses the VM terminology. But sure, I'm fine with it. I just thought it made sense to have the porting guide all use similar terminology.